### PR TITLE
[ActionSheet] Add color themer

### PR DESCRIPTION
**Pull request by codeman7**
### The problem

ActionSheet previously didn't have a color themer at all.

### The fix

This adds a color themer to ActionSheet, as well as test to test these use cases

### Related issues

#5039 

#### Swift
```
let actionSheet = MDCActionSheetController()
MDCActionSheetColorThemer.applySemanticColorScheme(colorScheme, to: actionSheet)
```

#### ObjC
```
MDCActionSheetController *actionSheet = [[MDCActionSheetController alloc] init];
[MDCActionSheetColorThemer applySemanticColorScheme:self.colorScheme
                            toActionSheetController:actionSheet];
```

### Screenshots
| Before | After |
| ------ | ----- |
|simulator screen shot - iphone x - 2018-09-24 at 09 17 13|simulator screen shot - iphone x - 2018-09-24 at 09 16 10|

_Screenshot generated with_
```
self.colorScheme.surfaceColor = .yellow
self.colorScheme.onSurfaceColor = .magenta
```

### CODE REVIEW

File: components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialActionSheet.h"
+#import "MaterialColorScheme.h"
+
+#import <Foundation/Foundation.h>
+
+@interface MDCActionSheetColorThemer : NSObject
```
Comment (jverkoey) [inline components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h]: Missing class docs.

File: components/ActionSheet/tests/unit/MDCActionSheetThemeTest.m [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  self.actionSheet = [[MDCActionSheetController alloc] init];
+  self.colorScheme = [[MDCSemanticColorScheme alloc] init];
+  UIColor *surface = UIColor.blueColor;
+  UIColor *onSurface = UIColor.redColor;
+  self.colorScheme.primaryColor = surface;
+  self.colorScheme.onPrimaryColor = onSurface;
+}
+
+#pragma mark - Header test
+- (void)testApplyColorThemerWithTitleAndMessage {
```
Comment (jverkoey) [inline components/ActionSheet/tests/unit/MDCActionSheetThemeTest.m]: [style]: empty newlines around method implementations here and throughout.

File: components/ActionSheet/tests/unit/MDCActionSheetThemeTest.m [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+#pragma mark - default test
+- (void)testApplyThemeToCells {
+  // When
+  [MDCActionSheetColorThemer applySemanticColorScheme:self.colorScheme
+                              toActionSheetController:self.actionSheet];
+  NSArray *cells = [MDCActionSheetTestHelper getCellsFromActionSheet:self.actionSheet];
+
+  // Then
+  for (MDCActionSheetItemTableViewCell *cell in cells) {
```
Comment (jverkoey) [inline components/ActionSheet/tests/unit/MDCActionSheetThemeTest.m]: Assert that cells is a non-empty array to catch empty loops passing this test.

File: components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h (hunk 1 above)
Comment (codeman7) [reply components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h]: I added the docs in #5209

Comment (jverkoey) [reply components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h]: Rad - the class itself also needs to have `/** */` docs for generated API docs.

Comment (codeman7) [reply components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h]: Oh yeah sorry I thought you meant the readme, I'll add those right now.

**Review by jverkoey — APPROVED** [review]

**Pull request merged**